index.d.ts

```diff
@@ -8,834 +8,369 @@ declare function Eris(token: string, options?: Eris.ClientOptions): Eris.Client;
-    user: { name: string; discriminator: string; id: string; avatar: string };
-  } & EmojiBase;
   interface PartialEmoji {
     id?: string;
     name: string;
   }
-  interface IntegrationOptions {
-    expireBehavior: string;
-    expireGracePeriod: string;
-    enableEmoticons: string;
-  }
-  interface PartialRole {
-    id?: number;
-    color?: number;
-    hoist?: boolean;
-    name?: string;
-    permissions?: number;
-    position?: number;
-    mentionable?: boolean;
-  }
-  interface PartialChannel {
-    id?: number;
-    type: number;
-    permission_overwrites?: Overwrite[];
-    name?: string;
-    topic?: string;
-    nsfw?: boolean;
-    bitrate?: number;
-    user_limit?: number;
-    rate_limit_per_user?: number;
-    parent_id?: number;
-  }
-  interface CreateGuildOptions {
-    region?: string;
-    icon?: string;
-    verificationLevel?: number;
-    defaultNotifications?: number;
-    explicitContentFilter?: number;
-    afkChannelID?: string;
-    afkTimeout?: number;
-    roles?: PartialRole[];
-    channels?: PartialChannel[];
+
+  interface MemberPartial {
+    id: string;
+    user: User;
   }
-  interface GuildOptions {
-    name?: string;
-    region?: string;
-    icon?: string;
-    verificationLevel?: number;
-    defaultNotifications?: number;
-    explicitContentFilter?: number;
-    systemChannelID?: string;
-    rulesChannelID?: string;
-    publicUpdatesChannelID?: string;
-    preferredLocale?: string;
-    afkChannelID?: string;
-    afkTimeout?: number;
-    ownerID?: string;
-    splash?: string;
-    banner?: string;
-    description?: string;
+  interface OldCall {
+    endedTimestamp?: number;
+    participants: string[];
+    region: string;
+    ringing: string[];
+    unavailable: boolean;
   }
   interface OldGuild {
-    name: string;
-    verificationLevel: 0 | 1 | 2 | 3 | 4;
-    splash?: string;
-    banner?: string;
-    region: string;
-    ownerID: string;
-    icon: string;
-    features: string[];
-    emojis: (Omit<Emoji, "user" | "icon"> & { available: boolean })[];
     afkChannelID?: string;
     afkTimeout: number;
-    mfaLevel: 0 | 1;
+    banner?: string;
+    emojis: (Omit<Emoji, "user" | "icon"> & { available: boolean })[];
+    explicitContentFilter: 0 | 1 | 2;
+    features: string[];
+    icon: string;
     large: boolean;
     maxPresences?: number;
-    explicitContentFilter: 0 | 1 | 2;
+    mfaLevel: 0 | 1;
+    name: string;
+    ownerID: string;
+    region: string;
+    splash?: string;
     systemChannelID?: string;
+    verificationLevel: 0 | 1 | 2 | 3 | 4;
   }
-  interface MemberOptions {
-    roles?: string[];
-    nick?: string;
-    mute?: boolean;
-    deaf?: boolean;
-    channelID?: string | null;
+  interface OldGuildChannel {
+    bitrate?: number;
+    name: string;
+    nsfw: boolean;
+    parentID?: string;
+    permissionOverwrites: Collection<PermissionOverwrite>;
+    rateLimitPerUser?: number;
+    position: string;
+    topic?: string;
```

Comment:
Discussed in #js_eris, topic can be null as well

Reply:
#981

Reply:
okay